### ISSUE TITLE
Azure 

### ISSUE THREAD

**Issue by tomerqueves** [hummingbot/hummingbot#5940]
## Why
The rationale behind this user story.

## What
Define or outline the task(s) that needs to be done.

## Definition of Done
Condition(s) or requirement(s) to be satisfied to mark this story as completed.

**Comment by rapcmia** [comment]
Hi @tomerqueves good day,
Can you please add the details for this ticket?

**Comment by tomerqueves** [comment]
no. sorry. i do not ride well github and think need not a ticket. if needed i'll erase the ticket. don't think so it's disturb.
